src/types.ts

```diff
@@ -1,7 +1,7 @@
 import type { Octokit } from "@octokit/core";
 import type { Endpoints } from "@octokit/types";
 
-export type TreeParameter = Endpoints["POST /repos/:owner/:repo/git/trees"]["parameters"]["tree"];
+export type TreeParameter = Endpoints["POST /repos/{owner}/{repo}/git/trees"]["parameters"]["tree"];
```

Comment:
I think we should add `@octokit/types` as a direct dependency of this package, as we use it in the code, that will make it more predictable. And I've seen `yarn` users having problems when using sub-dependencies

Reply:
will do!